_includes/next_meetup.html

```diff
@@ -0,0 +1,65 @@
+{% assign meetup_date = '15 de septiembre' %}
+{% assign meetup_time = '19 hs' %}
+{% assign meetup_place = 'Eagerworks | Juan Paullier 1218' %}
+{% assign meetup_first_speaker_name = 'Marcos Bellucci' %}
```

Comment:
Before merging, maybe we can put "To be confirmed" in each field

Reply:
or an if statement to not show the section until we are ready? (or both?) thinking that while we prepare a meetup we might have some things ready before everything is confirmed?

Reply:
I implemented both so we can have full control. Thanks for the suggestion.